langchainrb_rails.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "langchainrb", ">= 0.7", "< 0.19"
+  spec.add_dependency "langchainrb", ">= 0.7"
```

Comment:
@emk-klaay We should probably change this instead to be something like `"langchainrb", ">= 0.19"`